src/plugins/telemetryTable/components/table.vue

```diff
@@ -336,7 +337,8 @@ export default {
             markCounter: 0,
             paused: false,
             markedRows: [],
-            isShowingMarkedRowsOnly: false
+            isShowingMarkedRowsOnly: false,
+            hideHeaders: configuration.hideHeaders
```

Comment:
I think we can use, `hideHeaders` instead of `configuration.hideHeaders` as its updated in line 620?

Reply:
it needs to be initialized on mounted, thats why it gets it from configuration in the data prop

Reply:
I simplified the controls layout in the Inspector.